### PR TITLE
Replace ruby unlock script

### PR DESCRIPTION
**Pull request by ashchan**
This removes the `bitcoin_unlock.c` loading from the SDK, and uses the C script directly.

### CODE REVIEW

**Comment by ashchan** [comment]
Double checking tests.

File: Source/Utils/VerifyScript.swift [hunk 1]
```diff
@@ -17,7 +17,9 @@ final class VerifyScript {
     }
 
     private init() {
```
Comment (xxuejie) [inline Source/Utils/VerifyScript.swift]: Can we get rid of this and signal a proper compile error?

Comment (ashchan) [reply Source/Utils/VerifyScript.swift]: Sure. Will delete this class as it’s not intended to be used atm.

Comment (ashchan) [reply Source/Utils/VerifyScript.swift]: @xxuejie Deleted the VerifyScript.

**Pull request merged**